ksrates/plot_tree.py

```diff
@@ -4,27 +4,28 @@
 import ksrates.fc_manipulate_trees as fcTree
 import ksrates.fc_check_input as fcCheck
 import ksrates.fc_configfile as fcConf
+from ksrates.fc_rrt_correction import _ADJUSTMENT_TABLE
```

Comment:
Same comment for plot_paralogs.py: _ADJUSTMENT_TABLE is the only "constant" that I need to import because it doesn't come from a module that is actually used in plot_tree.py. This way I avoid importing the whole module fc_rrt_correction.py for just one variable.

Reply:
Alright.